### PR TITLE
First pass and polishing the CDA API docs

### PR DESCRIPTION
**Pull request by ChrisChinchilla**

### CODE REVIEW

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 1]
```diff
@@ -683,9 +657,9 @@ Order items by specifying your `order` parameter along with your attributes(`att
 
 ## Limit [/spaces/{space_id}/entries?access_token={access_token}&limit={value}]
 
-Specify the maximum number of results as limit search parameter.
+You can specify the maximum number of results as a `limit` search parameter.
 
-Please note that the maximum number of entries returned by the API is 1000. Limits higher than 1000 will be ignored. The default number of entries returned by the API is 100.
+**Note**: The maximum number of entries returned by the API is 1000 and it will ignore limits higher than **1000**. The default number of entries returned by the API is **100**.
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: Shouldn't the first 1000 be bold too?

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 2]
```diff
@@ -694,17 +668,17 @@ Please note that the maximum number of entries returned by the API is 1000. Limi
...
 
   + Attributes (Empty Array)
 
 ## Skip [/spaces/{space_id}/entries?access_token={access_token}&skip={value}]
 
-Specify an offset as the skip search parameter.
+You can specify an offset with the `skip` search parameter.
 
-By combining `skip` and `limit` you can achieve pagination:
+By combining `skip` and `limit` you can create pagination:
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: Create sounds to me like the wrong verb to me. How about:

> By combining `skip` and `limit` you can paginate through results:

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 3]
```diff
@@ -736,7 +710,7 @@ Assets can be filtered by their MIME type group by using the `mimetype_group` qu
 
 ### Query assets [GET]
 
-Example: Include only image assets.
+This example will include only image assets.
```
Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: "will return" would be more consistent with the previous example

File: apiary/_partials/sync-resources.apib [hunk 4]
```diff
@@ -1,14 +1,12 @@
...
 
-The Sync API allows you to keep a local copy of all content of a space up-to-date via delta updates.
-
-Learn more in the [developer center](https://www.contentful.com/developers/docs/concepts/sync/).
+The Sync API allows you to keep a local copy of all content in a space up-to-date via delta updates (only the content that has changed).
 
 ## Syncing specific content
 
-By default a sync includes all types: assets, entries of all content types and deletions.
+By default a sync request includes all resource types: assets, entries of all content types and deleted resources.
```
Comment (realityking) [inline apiary/_partials/sync-resources.apib]: Isn't it the response who includes the resource?

File: docs/tutorials/general/select-operator.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (realityking) [inline docs/tutorials/general/select-operator.md]: This file probably shouldn't be here.

File: apiary/cda.apib [hunk 6]
```diff
@@ -1,36 +1,38 @@
...
 
 ## Authentication
 
-To get content from the Content Delivery API, clients need to provide an access token. Access tokens are created per space and are used by clients to access the space's content.
-
-You can create access tokens for your own spaces in the "APIs" tab of the Contentful web app.
+To get content from the Content Delivery API, clients need to provide an access token. You create access tokens per space and use them in your clients to access a space's content.
 
-The Content Delivery API implements the standardized OAuth 2.0 bearer token specification. This authorization scheme is very simple and already supported by many HTTP clients.
+The API supports two mechanisms for supplying the access token. As an authorization HTTP `request-header` field and as an URI query parameter named `access_token`.
```
Comment (realityking) [inline apiary/cda.apib]: I don't understand the `request-header` part, it probably needs clarification,

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 7]
```diff
@@ -4,15 +4,15 @@
 
 ## HTTP details
 
-Learn more in the [developer center](https://www.contentful.com/developers/docs/references/http-details/).
+Learn more about the http details behind our delivery and preview APIs in the [developer center](https://www.contentful.com/developers/docs/references/http-details/).
 
 # Group Spaces
 
-All content in Contentful belongs to a space. You can think of spaces like databases, you will generally have at least one space for a project, but may opt to use a separate space for e.g. testing.
+All content entries in Contentful belongs to a space. You will generally have at least one space for a project, but may decide to use separate spaces for testing or staging purposes.
```
Comment (ChrisChinchilla) [inline apiary/_partials/delivery-and-preview-resources.apib]: @realityking Someone at sometime told me that we don't want to refer to Contentful as a 'database'… Tweaked the other point.

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 8]
```diff
@@ -58,13 +58,13 @@ Each space has a name, a set of locales, and some system metadata.
 
 :[What is a content type?](content-types-description.md)
 
-## Content type collection [/spaces/{space_id}/content_types?access_token={access_token}]
+## Content model [/spaces/{space_id}/content_types?access_token={access_token}]
```
Comment (ChrisChinchilla) [inline apiary/_partials/delivery-and-preview-resources.apib]: @realityking Exactly, let's maybe go with my hunch for now and see what happens…

Comment (realityking) [inline apiary/_partials/delivery-and-preview-resources.apib]: 👍

File: apiary/_partials/delivery-and-preview-resources.apib [hunk 9]
```diff
@@ -198,21 +194,19 @@ All of this, with examples, is documented in our [Images API](https://www.conten
...
 The maximum number of inclusions is `10`. The default number, if the parameter is not specified, is `1`. If you want no inclusions at all, you should set it to `0`.
 
-This feature is only available on the Content Delivery API and the Preview API.
+This feature is only available in the Content Delivery and Preview API.
 
-If linked entries have not already been retrieved in the standard `items` array, they will be placed in the `includes.Entry` array. Similarly, linked assets are put inside the `includes.Asset` array.
+If the standard `items` array has not already retrieved the linked entries, they will be in the `includes.Entry` array. Linked assets are inside the `includes.Asset` array.
 
-Note that only links between entries, spaces and assets are resolved. Links between spaces and content types are not included in the response when the `include` parameter is specified.
+**Note**: The `include` parameter only resolves links between entries, spaces and assets. Links between spaces and content types are not included in the response.
```
Comment (ChrisChinchilla) [inline apiary/_partials/delivery-and-preview-resources.apib]: @realityking This is the same comment as in the links concept document that I had approved by others. I did have to re-read this a few times before I understood, I'll try to improve.

**Pull request merged**